File: class_6/class_6/app/src/main/java/com/alejouribesanchez/class_6/Main2Activity.java
package com.alejouribesanchez.class_6;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.LinearLayout;

public class Main2Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main2);

        LinearLayout linearLayout = (LinearLayout)findViewById(R.id.activity_main2);
        linearLayout.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(Main2Activity.this, Main3Activity.class));
            }
        });
    }
}

File: class_6/class_6/app/src/main/java/com/alejouribesanchez/class_6/Main2Activity.kt
package com.alejouribesanchez.class_6

import android.content.Intent
import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.view.View
import android.widget.LinearLayout

class Main2Activity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main2)

        val linearLayout = findViewById(R.id.activity_main2) as LinearLayout
        linearLayout.setOnClickListener { startActivity(Intent(this@Main2Activity, Main3Activity::class.java)) }
    }
}
